hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/CarbonitingMetadataFreezerActor.scala

```diff
@@ -73,7 +73,7 @@ class CarbonitingMetadataFreezerActor(carboniterConfig: HybridCarboniteConfig,
     case Event(DatabaseUpdateCompleted(Success(_)), UpdatingDatabaseData(workflowId, _)) =>
       // All set; send 'complete' to CarboniteWorkerActor; wait for next workflow to carbonite
       carboniteWorkerActor ! CarboniteWorkflowComplete(workflowId)
-      goto(Pending)
+      goto(Pending) using NoData
```

Comment:
This can be removed as it is included in [this](https://github.com/broadinstitute/cromwell/pull/5242/files) PR.

Reply:
I think the merge should take care of it automatically